.github/workflows/push-tests.yml

```diff
@@ -1,10 +1,3 @@
-# ======================================
-# WARNING!
-# THIS FILE IS GENERATED FROM A TEMPLATE
-# DO NOT EDIT THIS FILE MANUALLY!
-# ======================================
-# The template is located in: push-tests.yml.j2
-
 name: Run tests on push
 on:
   push:
```

Comment:
Honestly, I think we should have this enabled also for RHEL branches. The issue is that we then need to use our self-hosted runners so this needs to be more enhanced / split. This is more like a future PR upgrade.